tiny/include/compile_options.h

```diff
@@ -0,0 +1,16 @@
+#ifndef COMPILE_OPTIONS_H_
+#define COMPILE_OPTIONS_H_
+
+// Add compiler
+#define TINY_COMPILER
```

Comment:
Should wrap these in `#ifndef`

Reply:
@hiperiondev @hiperiondev It makes sense to keep, but if people try to override these definitions via compiler flags (e..g `-DTINY_COMPILER`) their compiler will yell at them regarding the multiple definitions.

Wrapping them in `#ifndef` prevents this.

Reply:
Done